ACCESS_LIST.md

```diff
@@ -0,0 +1,171 @@
+[@DeeDeeG]: https://github.com/DeeDeeG
+[@Digitalone1]: https://github.com/Digitalone1
+[@softcode]: https://github.com/softcode589
+[@icecream17]: https://github.com/icecream17
+[@Spiker985]: https://github.com/Spiker985
+
+This document will serve the purpose of clearly defining who has access to what. This can be used as a way of knowing who within the org must be contacted for access or changes to certain pieces of software, secrets, or any otherwise single access pieces of data.
+
+* [Core](#core)
+  - [GitHub](#github)
+  - [Cirrus Ci](#cirrus-ci)
+  - [Codacy](#codacy)
+  - [Codecov](#codecov)
+  - [Crowdin](#crowdin)
+  - [Open Collective](#open-collective)
+  - [Infisical](#infisical)
+  - [Bitwarden](#bitwarden)
+  - [Chocolatey](#chocolatey)
+  - [macOS Binary Signing Keys](#macos-binary-signing-keys)
+* [Backend](#backend)
+  - [Google Cloud](#google-cloud)
+  - [Digital Ocean](#digital-ocean)
+  - [Google Domains](#google-domains)
+  - [Google Search](#google-search)
+  - [Vultr](#vultr)
+* [Social](#social)
+  - [Mastodon](#mastodon)
+  - [Reddit](#reddit)
+  - [Discord](#discord)
+* [Other](#other)
+  - [Codeberg](#codeberg)
+
+# Core
+
+Administrative actions relating to the core of the editor.
+Such as the `pulsar` repository or any non-bundled core repositories.
+Or otherwise any applications used to fulfill the needs of the pulsar.
+
+## GitHub
+
+Specifically for the `pulsar` repo:
+
+* `pulsar-edit/core-admins`
+* Pulsar Organization Admins
+
+## Cirrus CI
+
+The permissions of the `pulsar` repository on GitHub should sync to Cirrus CI.
+
+## Codacy
+
+The permissions of the `pulsar` repository on GitHub should sync to Codacy.
+
+Except when Codacy is being used on another one of our repos than whatever team permissions apply to that repo will sync.
+
+## Codecov
+
+The permissions of the `pulsar` repository on GitHub should sync to Codecov.
+
+Expect when Codecov is being used on another on of our repos, than whatever team permissions apply to that repo will sync.
+
+## Crowdin
+
+* Owner: [@Meadowsys]
+* Manager: [@confused-Techie]
+
+## Open Collective
+
+* Admins:
+  - [@kaosine]
+  - [@Meadowsys]
+  - [@confused-Techie]
+
+## Infisical
+
+* Admins:
+  - [@confused-Techie]
+  - [@Daeraxa]
+
+## Bitwarden
+
+* Owner: [@confused-Techie]
+* Admin:
+  - [@Daeraxa]
+  - [@DeeDeeG]
+* Manager:
+  - [@Meadowsys]
+  - [@kaosine]
+
+## Chocolatey
+
+* [@confused-Techie]
+
+## macOS Binary Signing Keys
+
+- [@Meadowsys]
+
+# Backend
+
+Administrative actions related to our cloud services or the backend specifically.
+Such as the package backend, frontend, microservices, or DNS changes.
+
+## Google Cloud
+
+* [@confused-Techie]
+
+## Digital Ocean
+
+* Owner: [@confused-Techie]
+* Member: [@mauricioszabo]
+* Read Only: [@Digitalone1]
+
+## Google Domains
+
+* [@confused-Techie]
+
+## Google Search
+
+* [@confused-Techie]
+
+## Vultr
+
+* Owner: [@kaosine]
+* Member: [@confused-Techie]
+
+## Coveralls
```

Comment:
the fact that there isn't a newline after Coveralls lowkey bothers me >.>

Reply:
Meant to only include `Codecov` as I had them confused. Removed `Coveralls`